### ISSUE TITLE
README

### ISSUE THREAD

**Issue by mikeskydev** [mikeskydev/unity-openxr-extensions#4]
Explain motivation, how to install, document current features.

**Comment by Sandstedt** [comment]
Do you use this project as a package? Or what's the best approach to get these extensions up and running in your own project? Just drag n drop the files into your project?

Could help write some documentation and example scenes, but need to know hoe to get it up and running first. Would love to get this repo rolling in any way or form.

**Comment by mikeskydev** [comment]
Yes it's a UPM package, you can drop this repo directly into the `Packages` folder of a unity project to have it work as an editable embedded package. For end users I want to publish this on OpenUPM #5 as it has a lot of benefits for versioning.

 > example scenes

I also plan to publish my testing project once i've cleaned it up which would really help!
